cli.js

```diff
@@ -52,9 +56,10 @@ const run = options => {
   const keyword = options._[0];
   const language = options.language || 'js';
   const shouldOpen = options.open || false;
+  const locale = options.locale || 'en-US';
 
   if (keyword !== undefined && keyword.length) {
-    mdn({keyword, language, shouldOpen});
+    mdn({keyword, language, shouldOpen, locale});
```

Comment:
@fernahh Can you please break these into individual new lines? Just so it reads better.